src/LabLog/Views/Teacher/Index.cshtml

```diff
@@ -1,9 +1,13 @@
-﻿@{
-    ViewData["Title"] = "List of Rooms";
-}
+﻿@model UserModel
 
-
-<p>This page is the admin landing page.</p>
-<p>It will contain a list of rooms, as well as
-    the options to create a new room. Room editing and deleting will be done from
-    within each Room's view page.</p>
\ No newline at end of file
+  <div class="wrapper">
+    <form class="form-signin" action="/Teacher" method="POST">
+      <h2 class="form-signin-heading">Please login</h2>
+      <input type="text" class="form-control" name="username" placeholder="E-Number" required="True" autofocus="" />
+      <input type="password" class="form-control" name="password" placeholder="Password" required="True"/>      
+      <label class="checkbox">

```

Comment:
when using a label element, set the "for" attribute, which will bind a mouseclick to the text of the label to the form element it's associated with. There's also a tag helper for these too

Reply:
Whoops, yeah I those for the text inputs, but then deleted the label entirely in favour of a placeholder. Forgot about the radio box.

Reply:
Can't use the asp-for tag helper for the RememberMe checkbox, as there isn't a matching entry in the UserModel. Should there be?